### ISSUE TITLE
Can't enable/disable buttons from restricted environment

### ISSUE THREAD

**Issue by kemayo** [Stanzilla/WoWUIBugs#64]
`1x FrameXML\RestrictedExecution.lua:431: Call failed: FrameXML\RestrictedFrames.lua:659: attempt to call field 'Enable' (a nil value)`

9.0.5 seems to have removed the `Enable`/`Disable` methods from the basic `Frame`.

This means that a `Button` in the restricted environment cannot be enabled/disabled, because RestrictedFrames.lua creates a `Frame` so it can call its methods without risking them being overridden, but those methods no longer exist.

See: RestrictedFrames.lua lines 650:
```
LOCAL_CHECK_Frame.Disable(frame);
```
and 659:
```
LOCAL_CHECK_Frame.Enable(frame);
```

Presumably we need a `LOCAL_CHECK_Button` equivalent for these type-specific methods.

**Comment by Meorawr** [comment]
Still affects 9.1.0.38312.

**Comment by Meorawr** [comment]
This should be fixed in an upcoming 9.1 PTR build.

**Comment by Meorawr** [comment]
Fixed in 9.1.0.38549, I'll close this now just to get it off the radar and reduce any confusion should Blizzard try to fix an already fixed bug.
